insights/core/exceptions.py

```diff
@@ -81,10 +84,20 @@ def __init__(self, content_type):
 
 class SkipException(SkipComponent):
     """
+    .. warning::
+        This class is deprecated, please use
+        :py:class:`insights.core.exceptions.SkipComponent` instead.
+
     Exception that should be thrown from parsers that are explicitly
     written to look for errors in input data.  If the expected error
     is not found then the parser should throw this exception to
     signal to the infrastructure that the parser's output should not be
     retained.
     """
-    pass
+    def __init__(self):
+        deprecated(
+            SkipComponent,
+            "Please use the :class:`insights.core.exceptions.SkipComponent` instead.",
+            "3.2.25"
+        )
+        super(SkipComponent, self).__init__()
```

Comment:
The same comment as above.
And the `Exception Message` should also be handled here, in case the customized `msg` is passed in.  e.g.`SkipException(msg)`

Reply:
I added handling the msg, I missed that sorry about that.